README.md

```diff
@@ -424,11 +424,11 @@ client.recordBreadcrumb("hello world")
     .withCustomData(someData);
 ```
 
-**Dont do this in a production environment:** You can set the factory to have the source location (class, method, line) added to the breadcrumb:
+You can set the factory to have the source location (class, method, line) added to the breadcrumb:
 ```java
 RaygunClientFactory factory = new RaygunClientFactory("YOUR_APP_API_KEY").withBreadcrumbLocations()
 ```
-While this can be incredibly useful for debugging it is **very resource intensive and will cause performance degredation**.
+While this can be incredibly useful for debugging it is **very resource intensive and will cause performance degredation**. We reccomend that you **do not do this in production**.
```

Comment:
```suggestion
While this can be incredibly useful for debugging it is **very resource intensive and will cause performance degradation**. We recommend that you **do not do this in production**.
```